spec/support/job_shared_examples.rb

```diff
@@ -0,0 +1,80 @@
+shared_examples 'background job' do
+  include ActiveJob::TestHelper
+  ActiveJob::Base.queue_adapter = :test
```

Comment:
This is good, but I'd prefer it if all the test configuration was in `spec_helper.rb`.